unstructured/ingest/runner/sharepoint.py

```diff
@@ -1,69 +1,66 @@
 import hashlib
 import logging
-import typing as t
 
-from unstructured.ingest.interfaces import PartitionConfig, ReadConfig
 from unstructured.ingest.logger import ingest_log_streaming_init, logger
 from unstructured.ingest.processor import process_documents
+from unstructured.ingest.runner.base_runner import Runner
 from unstructured.ingest.runner.utils import update_download_dir_hash
 from unstructured.ingest.runner.writers import writer_map
 
 
-def sharepoint(
-    verbose: bool,
-    read_config: ReadConfig,
-    partition_config: PartitionConfig,
-    site: str,
-    client_id: str,
-    client_cred: str,
-    files_only: bool,
-    path: str,
-    recursive: bool,
-    writer_type: t.Optional[str] = None,
-    writer_kwargs: t.Optional[dict] = None,
-    **kwargs,
-):
-    writer_kwargs = writer_kwargs if writer_kwargs else {}
+class Sharepoint(Runner):
```

Comment:
```suggestion
class SharePoint(Runner):
```
for consistency with naming [here](https://www.microsoft.com/en-us/microsoft-365/sharepoint/collaboration)?